bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/transform/UpConverter.scala

```diff
@@ -138,7 +138,38 @@ case class LastStepConverter(semester: Semester) extends SemesterConverter {
  * This converter will upgrade to 2018B.
  */
 case object SemesterConverter2018ATo2018B extends SemesterConverter {
-  override val transformers = Nil
+  val dssiSite: TransformFunction = {
+    case <dssi>{ns @ _*}</dssi> =>
+      object DssiSiteTransformer extends BasicTransformer {
+        override def transform(n: xml.Node): xml.NodeSeq = n match {
+          case p @ <Dssi>{q @ _*}</Dssi> => <Dssi id={p.attribute("id")}>{q.map(transform) +: <site>{Site.GS.name}</site>}</Dssi>
+          case <name>{name}</name>       => <name>DSSI {Site.GS.name}</name>
+          case elem: xml.Elem            => elem.copy(child = elem.child.flatMap(transform))
+          case _                         => n
+        }
+      }
+      StepResult("DSSI proposal has been assigned to Gemini South. Use ʻAlopeke instead for Gemini North.", <dssi>{DssiSiteTransformer.transform(ns)}</dssi>).successNel
```

Comment:
Why would you say `use Alopeke` when converting DSSI?

Reply:
From conversations, I was under the impression that Alopeke was being offered as a "better" DSSI, but I might be wrong. I'll double check that with Andy.

Reply:
Okay, talking to @andrewwstephens, it looks like Alopeke in speckle mode is an enhanced DSSI, so DSSI and GN proposals should migrate to that instead, with a message being issued. I'll make that change.